packages/document/docs/zh/guide/start/playground.mdx

```diff
@@ -0,0 +1,99 @@
+# Playground
+
+## 介绍
+
+Playground 是 Rsdoctor 提供的在线分析工具，支持上传构建分析数据并在线查看分析报告。无需本地安装 Rsdoctor，即可快速体验构建分析功能。
+
+## 支持的数据格式
+
+### 1. Rsdoctor 分析数据
+
+支持上传 Rsdoctor 生成的 JSON 格式分析数据，重新展示完整的分析报告。
+
+#### 配置要求
+
+需要配置 `output.mode: 'brief'` 并设置 `output.options.type` 为 `['json']` 或 `['html', 'json']`：
+
+> 具体配置可查看 [output 配置](/config/options/output)。
+
+```js
+// rspack.config.js 或 webpack.config.js
+module.exports = {
+  plugins: [
+    new RsdoctorRspackPlugin({
+      output: {
+        mode: 'brief',
+        options: {
+          type: ['json'], // 或 ['html', 'json']
+        },
+      },
+    }),
+  ],
+};
+```
+
+#### 使用步骤
+
+1. 构建项目后，在输出目录中找到 `rsdoctor-data.json` 文件
+2. 访问 [Playground 页面](https://rsdoctor.rs/preview/#/resources/uploader)
+3. 上传 `rsdoctor-data.json` 文件
+4. 查看完整的分析报告
+
+:::tip
+
+Brief 模式与 Normal 模式的区别：
+
+- **Brief 模式**：生成单一文件（HTML 或 JSON），便于分享和上传
+- **Normal 模式**：生成包含多个文件的文件夹，适合本地查看
+
+:::
+
+### 2. Rspack Stats 数据
+
+支持上传 Rspack 或 webpack 的 `stats.json` 文件，自动生成分析报告。
+
+#### 配置要求
+
+需要配置详细的 stats 选项以获取完整的构建信息：
+
+**方式一：使用 verbose 模式**
+
+```js
+module.exports = {
+  stats: 'verbose',
+};
+```
+
+**方式二：自定义 stats 配置**
```

Comment:
you have enabled 9 fields in the configuration below, but as a user, i don't know if all 9 fields are required? can I delete one of them? is it useful to add other fields?

Reply:
i think you still need to let user know

1. what field must be set to `true` (not documented yet)
2. what's the difference with method 1
3. how to choose between the two methods.